### PR TITLE
Account management new

### PR DESCRIPTION
**Pull request by Carla-Moz**
This is a hefty PR and I apologize in advance. I promise future PRs won't be so large.

Major changes include the following:

1. Centered container for all pages
2. Navbar and footer with routes to pages and external links 
3. Sign-out functionality in drop-down of navbar 
4. New account management styles which will probably change so let's keep as placeholder for now
5. Added Sass to the platform; there are still some lingering in line styles that need cleaning up as the sass files evolve. Might also be some repeating styles which I'll clean in next PR.
6. Not responsive yet; that's for the next PR.

### CODE REVIEW

**Review by rhelmer — CHANGES_REQUESTED** [review]
lgtm overall, I didn't focus on the style changes too much (except verifying that it looks and works ok visually), just a few minor things I noticed.

File: firebase.config.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
[the remark this answers is not in the document]
Comment (Carla-Moz) [reply firebase.config.json:1]: Yeah, this was accidentally removed from the static folder. Good catch. Thanks

File: src/app.html [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+    <link rel="stylesheet" href="/rally.css"/>
+    <link rel="stylesheet" href="/css/styles.css"/>
     <link
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
       integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
       crossorigin="anonymous"
       referrerpolicy="no-referrer"
-    />
+/>
```
[the remark this answers is not in the document]
Comment (Carla-Moz) [reply src/app.html:17]: Oh prettier must have formatted it that way.

File: src/lib/components/auth-cards/LaunchCard.svelte [hunk 3]
```diff
@@ -74,8 +78,11 @@
         background="transparent !important"
         borderColor="#000000"
         custom="card-button"
-        on:click={async () => {
-          await store.loginWithGoogle();
+        on:click={() => {
+          handleShowSpinner(true)
+          setTimeout(() => {
+            store.loginWithGoogle();
+          }, 1000);
```
[the remark this answers is not in the document]
Comment (Carla-Moz) [reply src/lib/components/auth-cards/LaunchCard.svelte:85]: Hm, not sure, maybe I was playing around with seeing the spinner a little but longer before signing in. So unnecessary haha.

File: src/lib/views/account-settings/UpdatePassword.svelte [hunk 4]
```diff
@@ -29,12 +28,12 @@
   let passwordErr = false;
   let passwordErrText;
 
-  console.log("store", store)
+  console.log("store", store);
```
[the remark this answers is not in the document]
Comment (Carla-Moz) [reply src/lib/views/account-settings/UpdatePassword.svelte:31]: Ack forgot to clean those up. Thanks

File: src/routes/_Sidebar.svelte [hunk 5]
```diff
@@ -102,11 +102,11 @@
           sveltekit:prefetch>Account Settings</a
         >
       </li>
-      <li in:fly={{ duration: 800, delay: 400, x: -15 }}>
+      <!-- <li in:fly={{ duration: 800, delay: 400, x: -15 }}>
```
[the remark this answers is not in the document]
Comment (Carla-Moz) [reply src/routes/_Sidebar.svelte:105]: Yes, meant to remove.

File: src/routes/account-settings/index.svelte [hunk 6]
```diff
@@ -74,22 +69,24 @@
   //   }
 
   const displayCard = (event) => {
+    console.log("event", event.detail.text);
```
[the remark this answers is not in the document]
Comment (Carla-Moz) [reply src/routes/account-settings/index.svelte:72]: Yup

File: src/stories/components/account/StorySettingsReadOnly.svelte [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+    leaveModal = true;
+  }}
+  class="settings-readonly"
+>
+  <div class="settings-readonly--first">
+    <h3 class="readonly-title">Contact Info</h3>
+    <div
+      class="readonly-email"
+      on:click={() => {
+        handleSelect(listArr[0]);
```
[the remark this answers is not in the document]
Comment (Carla-Moz) [reply src/stories/components/account/StorySettingsReadOnly.svelte:52]: Yes, true. Was originally an array of objects. Not sure why I changed it. Thanks.

**Review by stansky — APPROVED** [review]

**Pull request merged**